docs/data/base/components/select/UseSelect.tsx

```diff
@@ -142,23 +148,46 @@ interface Props {
   placeholder?: string;
 }
 
+interface OptionProps {
+  children?: React.ReactNode;
+  className?: string;
+  value: string;
+}
+
 function renderSelectedValue(value: string | null, options: SelectOption<string>[]) {
   const selectedOption = options.find((option) => option.value === value);
 
   return selectedOption ? `${selectedOption.label} (${value})` : null;
 }
 
+function CustomOption(props: OptionProps) {
+  const { children, value, className } = props;
+  const { getRootProps, highlighted } = useOption({
+    value,
+    disabled: false,
```

Comment:
I would do either:

1. Not add this at all
```suggestion
```
2. Use the value from the props (we can initialize it above to false):
```suggestion
    disabled,
```

This is closer to what developers would actually need.